cmflib/cmfquery.py

```diff
@@ -350,6 +350,34 @@ def get_pipeline_names(self) -> t.List[str]:
         """
         return [ctx.name for ctx in self._get_pipelines()]
 
+    def _transform_to_artifacts_dataframe(self, node):
```

Comment:
we need to use the  _transform_to_dataframe function here. If that function has any limitations, lets enhance that function instead of duplicating it

Reply:
Done